test/integration/multiplexed_upstream_integration_test.cc

```diff
@@ -337,7 +337,7 @@ TEST_P(MultiplexedUpstreamIntegrationTest, ManyLargeSimultaneousRequestWithBuffe
   manySimultaneousRequests(1024 * 20, 1024 * 20);
 }
 
-TEST_P(MultiplexedUpstreamIntegrationTest, ManyLargeSimultaneousRequestWithRandomBackup) {
+TEST_P(MultiplexedUpstreamIntegrationTest, ManyLargeSimultaneousRequestWithBackup) {
```

Comment:
We could also just remove this one - I think random pause >> pause so maybe just have the one?
We could leave it pause for now for HTTP/3 with a note it's not (yet) random pause

Reply:
Will remove it (I don't think it's doing anything actually for the HTTP/3) as we parameterize it as downstreams = {H2} upstreams = {H2, H3} but in the H2-H3 case we just abort from the test.